spec/lib/identity_job_log_subscriber_spec.rb

```diff
@@ -29,4 +29,30 @@
       issuer: build(:service_provider).issuer,
     )
   end
+
+  it 'logs errors when exception occurs in job' do
+    expect(Rails.logger).to receive(:error).exactly(2).times do |log|
+      json = JSON.parse(log)
+
+      expect(json['name']).to be_in [
+        'enqueue.active_job', 'perform.active_job'
+      ]
+      expect(json['job_class']).to eq('AddressProofingJob')
+      expect(json.key?('trace_id'))
+      expect(json.key?('duration_ms'))
+      expect(json.key?('job_id'))
+      expect(json.key?('timestamp'))
+      expect(json['exception_message']).to eq 'invalid base64'
```

Comment:
let's use parens for consistency with the comparison above?

```suggestion
      expect(json['exception_message']).to eq('invalid base64')
```